cdk/lib/support-workers.ts

```diff
@@ -522,8 +524,51 @@ export class SupportWorkers extends GuStack {
       treatMissingData: TreatMissingData.BREACHING,
       threshold: 0,
     }).node.addDependency(stateMachine);
+
+    // begining of Ad lite
+
+    const adLiteMetricDuration = Duration.minutes(5);
+    const adLiteEvaluationPeriods = 144; // The number of 5 minute periods in 12 hours
+    const adLiteAlarmPeriod = Duration.minutes(
+      adLiteMetricDuration.toMinutes() * adLiteEvaluationPeriods
+    );
+
+    new GuAlarm(this, `NoAdLiteAcquisitionInPeriodAlarm`, {
+      app,
+      actionsEnabled: isProd,
+      snsTopicName: `alarms-handler-topic-${this.stage}`,
+      alarmName: `URGENT 9-5 - ${
+        this.stage
+      } support-workers No successful Ad-Lite checkouts in ${adLiteAlarmPeriod.toHumanString()}.`,
+      metric: new MathExpression({
+        label: "AllTierThreeConversions",
```

Comment:
A small naming thing I noticed:

```suggestion
        label: "AllGuardianAdLiteConversions",
```

Reply:
nice catch!

Reply:
@maloky thanks for fixing this 🙌  I think the snapshot needs updating, are you happy to do that?